Documentation/config/core.txt

```diff
@@ -749,3 +749,9 @@ core.abbrev::
 	If set to "no", no abbreviation is made and the object names
 	are shown in their full length.
 	The minimum length is 4.
+
+core.wslcompat::
+	Tells git whether to enable wsl compatibility mode.
+	The default value is false. When set to true, git will set the mode
+	bits of the file in the way of wsl, so that the executable flag of
+	files can be set or read correctly.
```

Comment:
The Git project tends to use a lower-case `git` to refer to the executable, but Git with a capital "G" to refer to the program and the project. Therefore, I would like to see this:
```suggestion
	Tells Git whether to enable wsl compatibility mode.
	The default value is false. When set to true, Git will set the mode
	bits of the file in the way of wsl, so that the executable flag of
	files can be set or read correctly.
```

Also: this PR does more than that, doesn't it? It also sets UID/GID to 0...

Reply:
I write the wsl stat related functions for a few projects so I set uid/gid/mode. However, for git only, it's better to only set mode bits and that is allowed by MS. And this reduced the complexity of calculating the ea buffer.